fs/remote/resolver.go

```diff
@@ -355,6 +356,11 @@ func (f *fetcher) fetch(ctx context.Context, rs []region, retry bool, opts *opti
 	req.Header.Add("Range", fmt.Sprintf("bytes=%s", ranges[:len(ranges)-1]))
 	req.Header.Add("Accept-Encoding", "identity")
 	req.Close = false
+
+	// Recording the roundtrip latency for remote registry GET operation.
+	start := time.Now()
+	defer commonmetrics.MeasureLatency(commonmetrics.RemoteRegistryGet, start)
+	
```

Comment:
Yes, that's the intention. Thanks for the tip. This will be fixed with the next push.

Regarding adding layer label to latency metrics, I totally agree and plan to add it to that PR.